.github/workflows/locale_update_all.yaml

```diff
@@ -0,0 +1,60 @@
+name: "Locale Update All"
+
+on:
+  schedule:
+  - cron: '0 0 1,15 * *'
+  workflow_dispatch:
+permissions:
+  contents: read
+jobs:
+  locale_update_all:
+    if: github.repository_owner == 'ManageIQ'
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: manageiq/postgresql:13
+        env:
+          POSTGRESQL_USER: root
+          POSTGRESQL_PASSWORD: smartvm
+          POSTGRESQL_DATABASE: vmdb_i18n
+        options: --health-cmd pg_isready --health-interval 2s --health-timeout 5s --health-retries 5
+        ports:
+        - 5432:5432
+    env:
+      PGHOST: localhost
+      PGPASSWORD: smartvm
+      RAILS_ENV: i18n
+      SKIP_TEST_RESET: true
+      SKIP_DATABASE_SETUP: true
```

Comment:
I'm skipping TEST RESET and DATABASE_SETUP because I want to run bin/setup but not run these things because I only need the `vmdb_i18n` database created and migrated.  I don't need a test db, and certainly don't want to seed the database.